docs/known_issues.md

```diff
@@ -12,3 +12,13 @@ ViewComponent [isn't currently compatible](https://github.com/github/view_compon
 ## Inconsistent controller rendering behavior between Rails versions
 
 In versions of Rails < 6.1, rendering a ViewComponent from a controller doesn't include the layout.
+
+## Forms don't use the default form builder
+
+Calls to form helpers such as `form_with` in ViewComponents [don't use the default form builder](https://github.com/github/view_component/pull/1090#issue-753331927). This is by design, as it allows global state to change the rendered output of a component. Instead, consider passing a form builder into form helpers via the `builder` argument:
```

Comment:
Agree, the phrasing is awkward.

```suggestion
Calls to form helpers such as `form_with` in ViewComponents [don't use the default form builder](https://github.com/github/view_component/pull/1090#issue-753331927). This is by design, as it would otherwise allow global state to change the rendered output of a component. Instead, consider passing a form builder into form helpers via the `builder` argument:
```